pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "JPype1"
+version = '1.5.0.dev0'
+authors = [
+    {name = "Steve Menard", email = "devilwolf@users.sourceforge.net"},
+]
+maintainers = [
+    {name = "Luis Nell", email = "cooperate@originell.org"},
```

Comment:
do we still have this in current metadata? Luis retired like 8 years ago from this position :laughing:

Reply:
Yes. This can be an array (same for authors), and it is underfined what the distinction between authors and maintainers is. It could very reasonably be that we merge authors and maintainers into a single ``authors`` list - I don't know that the "maintainers" metadata is all that useful (seemingly not if it is out of date by 8 years...).

Reply:
This information is in our setup.py today https://github.com/jpype-project/jpype/blob/b603ff6ca6d58927262a202de83583a282924f2c/setup.py#L59.

I propose that we refresh this information _after_ moving to pyproject.toml